libwhisper/WhisperHost.h

```diff
@@ -50,36 +50,29 @@ class WhisperHost: public HostCapability<WhisperPeer>, public Interface, public
 public:
 	WhisperHost();
 	virtual ~WhisperHost();
-
-	unsigned protocolVersion() const { return 2; }
+	unsigned protocolVersion() const { return WhisperProtocolVersion; }
+	void cleanup(); ///< remove old messages
```

Comment:
I think we use this style around the codebase only for variables or class members. Or at least grepping around seems to show that. Please use `///` for function documentations.